### PR TITLE
Added widget scaffold and example of widget implementation.

### PR DESCRIPTION
**Pull request by azaslonov**

### CODE REVIEW

File: examples/widgets/conference-session/constants.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const widgetName = "conference-session";
+export const widgetDisplayName = "Conference session";
+export const widgetCategory = "Examples";
```
Comment (mikebudzynski) [inline examples/widgets/conference-session/constants.ts]: Rename the category to "Community"

File: examples/widgets/conference-session/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# External widget: Conference Session
```
Comment (mikebudzynski) [inline examples/widgets/conference-session/readme.md]: Change "External" to "Community"

File: scaffolds/widget/widgetModelBinder.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return contract.type === widgetName;
+    }
+
+    public canHandleModel(model: any): boolean {
+        return model instanceof WidgetModel;
+    }
+
+    public async contractToModel(contract: WidgetContract): Promise<WidgetModel> {
+        const model = new WidgetModel();
+        // model.property = contract.property;
```
Comment (ygrik) [inline scaffolds/widget/widgetModelBinder.ts]: maybe remove this comment or this is on purpose?

Comment (azaslonov) [reply scaffolds/widget/widgetModelBinder.ts]: yes, this is scaffold, so the comment is on purpose

Comment (mikebudzynski) [reply scaffolds/widget/widgetModelBinder.ts]: I think it's on purpose. It's good to have it and we follow the same in other files

File: examples/widgets/conference-session/readme.md (hunk 2 above)
Comment (azaslonov) [reply examples/widgets/conference-session/readme.md]: Done.

**Pull request merged**